internal/modulecontext/from_environment_test.go

```diff
@@ -12,8 +12,7 @@ import (
 func TestFromEnvironment(t *testing.T) {
 	ctx := log.ContextWithNewDefaultLogger(context.Background())
 
-	t.Setenv("FTL_POSTGRES_DSN_ECHO_ECHO", "postgres://echo:echo@localhost:5432/echo")
-
+	assert.NoError(t, SetDSNEnvar("echo", "echo", "postgres://echo:echo@localhost:5432/echo"))
```

Comment:
You should never use `os.Setenv()` in a test, use `t.Setenv()` instead. `SetDSNEnvar()` could be changed to return `(key, value string)` and then used as `t.Setenv(SetDSNEnvar(module, name))`

Reply:
Oh I think you commented at the same time as I did :) - yeah ^^ should work

Reply:
simultaneous comments 🤝